2023.md

```diff
@@ -136,3 +136,4 @@
 ## October (02)
 * [Try Pushed Authorization Requests (PAR) in Asgardeo](https://yoshani-malinka.medium.com/try-pushed-authorization-requests-par-in-asgardeo-dbabd9606806) by [Yoshani Ranaweera](https://yoshani-malinka.medium.com/)
 * [What is smoke testing?](https://medium.com/@asekaw/what-is-smoke-testing-4f0c4b180b3d) by [Aseka Warnakulasooriya](https://medium.com/@asekaw)
+* [Decoding Asgardeo User Types: A Practical Overview](https://inthiraj1994.medium.com/decoding-asgardeo-user-types-a-practical-overview-c554a20e7340) by [Tharmakulasingham Inthirakumaaran](https://medium.com/@inthiraj1994)
```

Comment:
Shall we update the blog count for October as well?

Reply:
Updated